### ISSUE TITLE
DSM 6b2 can't run couch potato anymore

### ISSUE THREAD

**Issue by BrianGilbert** [SynoCommunity/spksrc#2104]
tail of /var/log/messages when I try to run it:

```
Jan 26 19:37:46 DiskStation [262875.768526] init: pkgctl-couchpotatoserver-custom pre-start process (24289) terminated with status 1
Jan 26 19:37:46 DiskStation synoscgi_SYNO.Core.Package.Control_1_start[23935]: servicecfg_internal_lib.c:181 Failed to start job [pkgctl-couchpotatoserver-custom][0xD900 manager.cpp:204]
Jan 26 19:37:48 DiskStation synoscgi_SYNO.Core.Package.Control_1_start[23935]: service_initjob_action.c:27 InitJobStart failed, [0xD900 manager.cpp:204]
Jan 26 19:37:48 DiskStation synoscgi_SYNO.Core.Package.Control_1_start[23935]: pkgstartstop.cpp:246 Failed to start pkgctl-couchpotatoserver-custom (err=-1) [0xD900 manager.cpp:204]

```

**Comment by danfozzy** [comment]
Same for sabnzbd and lazylibrarian.

**Comment by Hubfront** [comment]
I upgraded from 5.2 to 6.0 beta 2 and did leave sabnzbd-testing 8.0 beta 5 installed. It's still running without problems. I uninstalled the release version 7.x before upgrading though.

**Comment by BrianGilbert** [comment]
I was able to get it running by changing perms as per https://github.com/SynoCommunity/spksrc/issues/2104#issuecomment-176953290

**Comment by DrSeussFreak** [comment]
I just tried this @BrianGilbert , it does not work for me., which file did you modify for RUNAS?  I grepped all of the files and only found 1 reference (which seems right) in "/usr/local/couchpotatoserver/share/CouchPotatoServer" and that file was "CouchPotato.py"

**Comment by DrSeussFreak** [comment]
So I have it up and running now, but I am no longer using the SynoCommunity package.  I had a similar issue early last year with Mylar, and I ran it by manually installing it.

First I ran "git clone https://github.com/RuudBurger/CouchPotatoServer -b develop" while in /usr/local.  Then I ran a "chown -R auto:users CouchPotatoServer".    Last I ran "nohup python /usr/local/CouchPotatoServer/CouchPotato.py < /dev/null &" to start it, and then connected on port 5050 (the default).  Now I am up and running.

**Comment by BrianGilbert** [comment]
@DrSeussFreak 

For the runs value I edited /var/packages/[appname]/scripts/start-stop-status

**Comment by daurnimator** [comment]
I changed start line in `/var/packages/[appname]/scripts/start-stop-status` from using `su - ${USER} -c` to `sudo -u ${USER}`

**Comment by simenon** [comment]
Modify `/var/packages/[appname]/scripts/start-stop-status` so it shows as below, and it works on DSM 6.0-7321

```
start_daemon ()
{
    sudo -u ${USER} ${PYTHON} ${COUCHPOTATOSERVER} --daemon --pid_file ${PID_FILE} --config_file ${CFG_FILE}
}
```

**Comment by MrBLJ** [comment]
@simenon Sir ... you solved my problem ! I salute you :+1:  !!!

**Comment by bull5-i** [comment]
@simenon Thanks, this worked for me too!

**Comment by DidierLoiseau** [comment]
I had the same issue installing Couchpotato from the package 2 days ago, and I came up with the same solution as @daurnimator.

Any idea why `su` does not work with the `couchpotatoserver-custom` user? It works with the `sickbeard-custom` user here, so what's the difference?

Note that [some people suggest to run it as root](https://forum.synology.com/enu/viewtopic.php?t=117015) (maybe unknowingly), but not using `su`.

BTW wasn't PR #6272 supposed to fix it? Or was there no release of the package since then?

**Comment by BrianGilbert** [comment]
I'm running it via docker now instead of messing with perms.

**Comment by Safihre** [comment]
Closing due to age. Test-packages for DSM6.1 and new architectures available here: #3138.
Those will later be published to the real package rep.
